CMakeLists.txt

```diff
@@ -634,9 +635,9 @@ install(PROGRAMS ${CMAKE_BINARY_DIR}/git-cvsserver  #${CMAKE_SOURCE_DIR}/gitk-gi
 		DESTINATION bin)
 
 list(REMOVE_ITEM PROGRAMS_BUILT git git-shell)
-		install(TARGETS ${PROGRAMS_BUILT}
-				RUNTIME DESTINATION libexec/git-core)
-				
+install(TARGETS ${PROGRAMS_BUILT}
+		RUNTIME DESTINATION libexec/git-core)
+
```

Comment:
This indentation fix should probably be squashed into the earlier commit that introduced the indentation problem.

I usually fire up an interactive rebase in those instances,
1. replacing the `pick` with an `edit` command for the commit that fixed the indentation.
2. Then, when the interactive rebase stops at that commit, I use `git checkout -p HEAD^` to _revert_ the fix.
3. I then commit it with `git commit -m TO-DROP`, so that I will remember to drop it during a subsequent interactive rebase.
4. Then re-revert it with `git revert -n HEAD` and 
5. find the commit which introduced the problem (in this case, `git log -L 638,640:CMakeLists.txt` would help me figure that out), and
6. `git commit --fixup <that-commit>`
7. Then I continue the interactive rebase and let it complete: `git rebase --continue`
8. Finally, I run another interactive rebase (with `--autosquash`, or more correctly, with `rebase.autosquash=true` configured in my user-wide config) which automatically handles that `fixup!` commit, and I still have to replace the `pick` in front of the `TO-DROP` by a `drop` command.

It is surprising to me how involved this looks when I write it up; I am doing this _so often_ that it is almost second nature to me.